spec/presenters/single_content_item_presenter_spec.rb

```diff
@@ -230,7 +230,8 @@
         content_id: 'content-id',
         publishing_app: 'whitehall',
         base_path: '/the/base/path',
-        document_type: 'news_story'
+        document_type: 'news_story',
+        parent_content_id: nil
```

Comment:
I think the value here should be consistent with the default value. If the default value is `''` then I don't think we can ever have `nil` here right?

Reply:
Eek good spot.

Reply:
Fixed